nimbus.fml.yaml

```diff
@@ -33,6 +33,25 @@ features:
             "contile-top-sites": false,
           }
         }
+  onboardingcreen:
+    description: The homescreen that the user goes to when they press home or new tab.
+    variables:
+      sections-enabled:
+        description: "This property provides a lookup table of whether or not the given section should be enabled."
+        type: Map<OnboardingScreenSection, Boolean>
+        default:
+          {
+            "jump-back-in-cfr": true,
```

Comment:
Do we need this "jump-back-in-cfr"? I don't see it used in this PR, but not sure if there's another requirement somewhere.